Makefile

```diff
@@ -106,6 +106,23 @@ _docker-%: build-all-platforms
 build-docker-image: _docker-build
 push-docker-image: _docker-push
 
+export COSIGN_REPOSITORY?=ghcr.io/jetstack/jetstack-secure/cosign
+export COSIGN_EXPERIMENTAL=1
+
+.PHONY: sign-docker-image
+sign-docker-image:
+	@cosign sign $(DOCKER_IMAGE):$(VERSION)
+
+.PHONY: sbom-docker-image
+sbom-docker-image:
+	@syft $(DOCKER_IMAGE):$(VERSION) -o cyclonedx > bom.xml
```

Comment:
I guess there's no need to create two separate SBOMs (one for source and one for base image) because `syft` is managing to pickup all the Go dependencies within the compiled image?

Reply:
Yep, you'd just end up with duplication.

Reply:
As long as we are sure that this will pick up all dependencies?

Reply:
```
$ pwd
/home/ribbybibby/go/src/github.com/jetstack/jetstack-secure

$ syft dir:. | grep -v go-module
 ✔ Indexed .
 ✔ Cataloged packages              [128 packages]
NAME                                                       VERSION                               TYPE
```
^ the only dependencies that it picks up from the source are go-modules which are also extracted when scanning the container image

Reply:
was just sanity checking - awesome 😄